examples/concatenate.py

```diff
@@ -13,14 +13,18 @@
 #                                                     existing kwarg! (dimname, dimlabel,
 #                                                     etc -- may also not be a kwarg, but
 #                                                     positional)
+print("concatenate without sort:")
+print(combined)
 combined.sort(
     "t"
 )  # sort is already applied in function but included here for clarity
+print("concatenate after sort:")
 print(combined)
 # }}}
 # {{{ when we make a new dimension - demos the existing functionality -- see above about kwarg
 data1 = psd.nddata(r_[0:4:4j], "t")
 data2 = psd.nddata(r_[0:4:4j], ["t"]) * 1.3
 combined = psd.concat([data1, data2], dimname="indirect")
+print("concatenate along new dimension:")
```

Comment:
I didn't want to create a huge diff, but please move this file to `examples_undoc`